src/test/msgr/test_msgr.cc

```diff
@@ -721,7 +721,6 @@ TEST_P(MessengerTest, AuthTest) {
   }
   ASSERT_TRUE(conn->is_connected());
   ASSERT_EQ(1U, static_cast<Session*>(conn->get_priv().get())->get_count());
-
```

Comment:
this is just a removed line from this file. If this line is still the same you had in the previous revision (it was basically an added space in the line, IIRC), you can simply grab the file from master and copy it to your branch; then commit the changes, and once squashed against the current patch set, you should end with no changes at all (well, assuming that test hasn't been changed before, but that can be resolved by rebasing on top of master).

Reply:
I am checking on this. For some reason after squeezing the commits and rebasing it on top of master, this was the end result, which I didn't expect. Looking into it.